components/brave_wallet_ui/components/desktop/popup-modals/index.tsx

```diff
@@ -24,6 +24,7 @@ export interface Props {
   headerPaddingVertical?: number
   headerPaddingHorizontal?: number
   borderRadius?: number
+  fullHeight?: boolean
```

Comment:
Let's use a more versitle height prop

```suggestion
  height?: string
```

Reply:
Updated!